bean/internal/driver/datasource/session/session.go

```diff
@@ -119,10 +123,14 @@ func (ds *dataSource) Delete(
 	ctx context.Context,
 	id string,
 ) error {
-	err := ds.cache.Client.Del(ctx, id).Err()
+	err := ds.cache.Client.Del(ctx, wrap(id)).Err()
 	if err != nil {
 		return fmt.Errorf("failed to delete session from cache: %w", err)
 	}
 
 	return nil
 }
+
+func wrap(id string) string {
+	return fmt.Sprintf("%s:%s", key, id)
+}
```

Comment:
This is bad, it can easily be forgotten

Redis should have methods which apply the namespace prefix, instead

Reply:
31e6529cb2c0c56cab2b5e629ee07719d7f738ce